### PR TITLE
Update electronic-prescription-service-api.yaml

### PR DESCRIPTION
**Pull request by CharlotteBunker**
Updated most text above Endpoints and moved error messages so they sit within each endpoint.

### CODE REVIEW

File: specification/electronic-prescription-service-api.yaml [hunk 1]
```diff
@@ -13,77 +13,75 @@ info:
...
-    * cancel a tertiary care (homecare) prescription
-
-    ### As a dispenser
-    You can:
+     ### Prescribers in primary and secondary care can:
+    * create a prescription
+    * encode data so the prescription is ready to sign
+    * cancel a prescription
+    
+    Create and cancel a prescription are both done by the prescriber. Encoding data so the prescription is ready to sign is done by the prescribing system.
```
Comment (dan-jones-iw) [inline specification/electronic-prescription-service-api.yaml:23]: `Creating and cancelling`?

File: specification/electronic-prescription-service-api.yaml [hunk 2]
```diff
@@ -218,10 +209,13 @@ paths:
   /FHIR/R4/$prepare:
     post:
       operationId: prepare-prescription
-      summary: Encode prescription data
+      summary: Encode prescription data so it's ready to sign - Prescribing
       description: |
         ## Overview
-        Use this endpoint to convert a FHIR [prescription-order message](https://simplifier.net/guide/NHSDigital-Medicines/Home/FHIRAssets/AllAssets/MessageDefinition-duplicate-2/prescription-order) into HL7 V3 signature fragments which can then be signed by the prescriber.
+        Use this endpoint to convert an unsigned FHIR prescription into the piece of data that needs to be digitall signed.
```
Comment (dan-jones-iw) [inline specification/electronic-prescription-service-api.yaml:215]: typo in digitally

File: specification/electronic-prescription-service-api.yaml [hunk 3]
```diff
@@ -270,16 +264,14 @@ paths:
   /FHIR/R4/$process-message#prescription-order:
     post:
       operationId: send-prescription-order-message
-      summary: Create a new prescription
+      summary: Create a new prescription - Prescribing
       description: |
         ## Overview
         Use this endpoint to create a new prescription in EPS.
+        
+        The prescription must include a digital signature, which can be generated using the Signing Service.
```
Comment (dan-jones-iw) [inline specification/electronic-prescription-service-api.yaml:272]: link to Signing Service docs?

File: specification/electronic-prescription-service-api.yaml [hunk 4]
```diff
@@ -218,10 +209,13 @@ paths:
   /FHIR/R4/$prepare:
     post:
       operationId: prepare-prescription
-      summary: Encode prescription data
+      summary: Encode prescription data so it's ready to sign - Prescribing
```
[the remark this answers is not in the document]
Comment (CharlotteBunker) [reply specification/electronic-prescription-service-api.yaml:212]: The tags don't seem to pull through to the live OAS page. By adding this text to the summary section, users can see at a glance in the table of contents what's relevant to them. If the tags should be functioning differently, then yeah the text is redundant. Either way, I just want users to quickly see which endpoints are relevant to them

File: specification/electronic-prescription-service-api.yaml [hunk 5]
```diff
@@ -95,35 +93,45 @@ info:
...
-    This API is [RESTful](https://digital.nhs.uk/developer/guides-and-documentation/our-api-technologies#basic-rest).
-
+ This API is [RESTful](https://digital.nhs.uk/developer/guides-and-documentation/our-api-technologies#basic-rest).
+ 
     It conforms to the [FHIR](https://digital.nhs.uk/developer/guides-and-documentation/our-api-technologies#fhir) global standard for health care data exchange, specifically to [FHIR R4 (v4.0.1)](https://hl7.org/fhir/r4/), except that it does not support the [capabilities](http://hl7.org/fhir/R4/http.html#capabilities) interaction.
-
-    It includes some country-specific FHIR extensions, which are built against [FHIR UK Core](https://digital.nhs.uk/services/fhir-uk-core).
-
+    
+    It includes some country-specific FHIR extensions, which are built against [FHIR UK Core, specifically [UKcore.stu1 0.5.1](https://simplifier.net/packages/fhir.r4.ukcore.stu1/0.5.1).
```
[the remark this answers is not in the document]
Comment (CharlotteBunker) [reply specification/electronic-prescription-service-api.yaml:100]: We want to be as specific as poss to cut down on the volume of questions that users have. However I appreciate that if the version changes frequently, then the OAS will need to as well. In which case I can pull together a process about what happens and when.

**Pull request merged**